internal/server/count/count.go

```diff
@@ -132,3 +128,62 @@ func Count(
 	}
 	return result, nil
 }
+
+// Count checks if entities have data for stat vars and stat var groups.
+//
+// Returns a two level map from sv/svg dcid to entity dcid to the number of
+// sv with data. For a given sv/svg, if an entity has no data, it will
+// not show up in the second level map.
+func Count(
+	ctx context.Context,
+	st *store.Store,
+	cachedata *cache.Cache,
+	svOrSvgs []string,
+	entities []string,
+) (map[string]map[string]int32, error) {
+	result, err := countInternal(ctx, st, cachedata, svOrSvgs, entities)
+	if err != nil {
+		return nil, err
+	}
+	// Check for count for computed observations.
+	// Use counts of formula variables as a heuristic.
+	for _, dcid := range svOrSvgs {
+		formulas, ok := cachedata.SVFormula()[dcid]
+		if !ok {
+			continue
+		}
+		for _, entity := range entities {
+			if _, ok := result[dcid][entity]; ok {
+				continue
+			}
+			for _, f := range formulas {
+				variableFormula, err := formula.NewVariableFormula(f)
+				if err != nil {
+					return nil, err
+				}
+				calculatedCount, err := countInternal(
```

Comment:
can you batch the countInternal call for all entities and variables?

Reply:
Done